automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +28,33 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
+    SKIPPED_TESTS=$(__get_skipped_tests)
+
     # Run KubeMacPool functional tests
     cd ${TMP_COMPONENT_PATH}
-    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" make functest
+
+    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml --ginkgo.skip $SKIPPED_TESTS" make functest
+}
+
+function __get_skipped_tests() {
+	local skipped_regex=""
+
+	if [ ! -z "$(__is_vm_opt_out)" ]; then
+		skipped_regex="${skipped_regex} \(opt-in\smode\)"
+	elif [ ! -z "$(__is_vm_opt_in)" ]; then
+		skipped_regex="${skipped_regex} \(opt-out\smode\)"
+	fi
+
+	echo $skipped_regex
+}
+
+function __is_vm_opt_in() {
+	grep 'default/mutatevirtualmachines_opt_in_patch.yaml' hack/components/bump-kubemacpool.sh
+}
+
+function __is_vm_opt_out() {
+	grep 'default/mutatevirtualmachines_opt_out_patch.yaml' hack/components/bump-kubemacpool.sh
```

Comment:
Could you please move this to code to another file so we could keep this script simple as possible?

Reply:
I was actually thinking about to move it inside the `__get_skipped_tests`.
It's simple enough and doesn't qualify a function of it's  own. + now with the comment it's well documented.

Tell me what you think